autoscaler/cluster.py

```diff
@@ -54,11 +54,8 @@ class Cluster(object):
     # just prioritize the ones that we're confident about
     _GROUP_DEFAULT_PRIORITY = 10
     _GROUP_PRIORITIES = {
```

Comment:
The flag type I added would make it easy to add this as a commandline flag . . .

;)